src/utils.js

```diff
@@ -69,34 +69,16 @@ export default {
 			})
 		);
 	},
-	editLog(data) {
-		const tempFilePath = "./.shortlog";
-
-		return new Promise((resolve, reject) => {
-			this.writeFile(tempFilePath, data);
-			editor(tempFilePath, code => {
-				if (code === 0) {
-					const contents = this.readFile(tempFilePath);
-					fs.unlinkSync(tempFilePath);
-					resolve(contents);
-				} else {
-					reject(`Unable to edit ${tempFilePath}`);
-				}
-			});
-		});
-	},
-	editBody(data) {
-		const tempFilePath = "./.pullrequest";
-
+	editFile(data, filePath) {
```

Comment:
We could probably get rid of `filePath` completely and generate that location...

`_.uniqueId('.editFile_');`

In our clean-up we'd need to remove (or unlinkSync) `.editFile_*`